test/conformance/api/v1/configuration_test.go

```diff
@@ -30,6 +30,45 @@ import (
 	v1test "knative.dev/serving/test/v1"
 )
 
+// TestConfigurationGetAndList tests Getting and  Listing Configurations resources.
+//   This test doesn't validate the Data Plane, it is just to check the APIs for conformance
+func TestConfigurationGetAndList(t *testing.T) {
+	t.Parallel()
+	clients := test.Setup(t)
+
+	names := test.ResourceNames{
+		Service: test.ObjectNameForTest(t),
+		Image:   test.PizzaPlanet1,
+	}
+
+	// Clean up on test failure or interrupt
+	test.EnsureTearDown(t, clients, &names)
+
+	// Setup initial Service
+	if _, err := v1test.CreateServiceReady(t, clients, &names); err != nil {
+		t.Fatalf("Failed to create initial Service %v: %v", names.Service, err)
+	}
+
+	list, err := v1test.GetConfigurations(clients)
```

Comment:
There's no test for `GetConfiguration` here, is that by design?

Reply:
@markusthoemmes there is no GetConfiguration because there is already a `func` called `fetchConfiguration` that is used in the next test, exercising the GET API. 
But I do agree that then the name of the test is misleading.. so I will add it to the test as well.

Reply:
@markusthoemmes done